Assets/FeatureVisualizeEnemies/VisualizeEnemies.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class VisualizeEnemies : MonoBehaviour {
+
+    private GameObject[] enemiesList;
+    public GameObject filtersSystem;
+    private FilterSelector filterSelector;
```

Comment:
The `FilterSelector` class doesn't exist, maybe do you mean `EnemyFilterSelector`?

Reply:
I was referring to the FilterSelector in the FiltersSystem feature, which shows the correct filter according to the mouse wheel input.